LICENCE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
@elsamuko Please rename file to LICEN**S**E

Reply:
Done